generic/tests/cfg/unattended_install.cfg

```diff
@@ -83,8 +83,22 @@
         - large_image:
             only unattended_install.cdrom
             only qcow2 qcow2v3
-            image_size_image1 = 2300G
-            image_size = 2300G
+            install_timeout = 7200
+            RHEL.6:
```

Comment:
there is an anaconda bug, when format disk on RHEL.6 with 16TB data disk, it will be error. So not add data disk for RHEL.6